__tests__/app.test.js

```diff
@@ -109,6 +109,59 @@ describe('GET commands', () => {
 })
 
 
+
+
+describe('PATCH commands', () => {
+    test('200 status: should update the votes value for a given review_id by changing it by the indicated number of votes in a passed object', () => {
+        return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_votes : 1})
+        .expect(200)
+        .then(({body})=>{
+            expect(body.updatedReview).not.toHaveProperty("inc_votes")
```

Comment:
I mean, you are right, this key _shouldn't_ be included, but it's not in the review structure determined in our db, so I don't think the line above is necessary

Reply:
It would be nice to check whether this has actually made it's way into the database. You can do this by utilising your getCommentsById endpoint and asserting that the response should reflect the changes to votes made when you did your patch.